### PR TITLE
feat(samples): Add Pnp Convention unit tests

### PR DESCRIPTION
**Pull request by bikamani**
### Changes include
- Add unit tests for Pnp Helper.
- Thermostat and Temperature controller should be modules of sample projects and not service.

### CODE REVIEW

File: device/iot-device-samples/pnp-device-sample/temperature-controller-device-sample/pom.xml [hunk 1]
```diff
@@ -23,6 +23,12 @@
             <version>2.11.2</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
```
Comment (barustum) [inline device/iot-device-samples/pnp-device-sample/temperature-controller-device-sample/pom.xml:29]: why do we need to add junit for the samples project?

**Review by barustum — APPROVED** [review]

File: service/pom.xml [hunk 2]
```diff
@@ -20,9 +20,6 @@
     <modules>
         <module>iot-service-client</module>
         <module>iot-service-samples</module>
-        <module>iot-service-samples/digitaltwin-service-samples</module>
```
Comment (bikamani) [inline service/pom.xml:23]: Note: samples shouldn't be included as modules here but in service-samples.

**Pull request merged**